staging/src/k8s.io/client-go/dynamic/client.go

```diff
@@ -166,9 +179,11 @@ func (rc *ResourceClient) Get(name string, opts metav1.GetOptions) (*unstructure
 		parameterEncoder = defaultParameterEncoder
 	}
 	result := new(unstructured.Unstructured)
+	resourceName, subresourceName := rc.parseResourceSubresourceName()
```

Comment:
Which subresources support List?

Reply:
I'm not aware of any based on [api discovery](https://github.com/kubernetes/kubernetes/pull/57171/files#diff-e79f254ff9fd86bebf69fdaa20843c2f). This change is in `Get` (github has some weird display problem :) ).

Reply:
Oops, yes. :)